tests/forms_tests/field_tests/test_emailfield.py

```diff
@@ -30,7 +30,7 @@ def test_email_regexp_for_performance(self):
         f = EmailField()
         # Check for runaway regex security problem. This will take a long time
         # if the security fix isn't in place.
-        addr = "viewx3dtextx26qx3d@yahoo.comx26latlngx3d15854521645943074058"
+        addr = "viewx3dtextx26qx3d@yahoo.comx26latlngx3d15854521645943074058.aa"
```

Comment:
Note to other reviewers: DomainNameValidator.tld_re doesn't allow digits in a top level domain, so the original addr here becomes invalid once EmailValidator borrows DomainNameValidator's logic. I'm not sure whether adding a valid TLD (".aa") impacts the check for "runaway regex security problem."